examples/distroless/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# ch-test-scope: standard
+# Distroless is a Google project providing slim images that contain runtime
+# dependencies only. https://github.com/GoogleContainerTools/distroless
+# The python3 image was chosen for ease of testing.
+FROM gcr.io/distroless/python3
+COPY . /distroless
+
+# Without the CMD directive ch-builder2tar will fail when using docker
+CMD ["/distroless/hello.py"]
```

Comment:
Interestingly once PR #698 is merged, this will become no longer needed.

Reply:
Would you like me to block this PR pending #698 ?

Reply:
No, that one will take a while.